CIScripts/JenkinsTesterDSC/JenkinsTester.ps1

```diff
@@ -0,0 +1,55 @@
+Configuration JenkinsTester {
+    param(
+        [Parameter(Mandatory=$true)][string[]]$MachineName
+    )
+
+    Import-DscResource -ModuleName PsDesiredStateConfiguration
+    Import-DscResource -ModuleName cChoco
+    Import-DscResource -ModuleName PowerShellModule
+
+    Node $MachineName {
+        WindowsFeature NetFrameworkCore {
+            Ensure    = "Present"
+            Name      = "NET-Framework-Core"
+        }
+
+        cChocoInstaller installChoco {
```

Comment:
Do you think this is necessary since cChoco should be already installed.

Reply:
cChoco will be installed but not choco

Reply:
ok

Reply:
I agree it is required.